src/EA.Iws.Web/Areas/NotificationMovements/Controllers/PrenotificationBulkUploadController.cs

```diff
@@ -11,5 +15,27 @@ public PrenotificationBulkUploadController(IMediator mediator)
         {
             this.mediator = mediator;
         }
+
+        [HttpGet]
+        public ActionResult UploadPrenotifications(Guid notificationId)
+        {
+            var model = new PrenotificationBulkUploadViewModel(notificationId);
+
+            return View(model);
+        }
+
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        public async Task<ActionResult> UploadPrenotifications(Guid notificationId, PrenotificationBulkUploadViewModel model)
```

Comment:
Please remove the async.

Reply:
async keyword removed and method return type changed to ActionResult